boards/bcm2836/Packages/build.txt

```diff
@@ -0,0 +1,19 @@
+@echo off
+
+REM Run this command from the IoTCoreShell prompt only
```

Comment:
Should we move this file to the other repo then? Since it is primarily signing drivers and building packages.

---
In reply to: [97693109](https://github.com/ms-iot/bsp/pull/33#discussion_r97693109) [](ancestors = 97693109)